splash/request_middleware.py

```diff
@@ -72,6 +72,59 @@ def process(self, request, render_options, operation, data):
         return request
 
 
+class ContentTypeMiddleware(object):
+    """
+    Response middleware, aborts responses depending on the content type.
+    A response will be aborted (and the underlying connection closed) after
+    receiving the response headers if the content type of the response is not
+    in the whitelist or it's in the blacklist. Both lists support wildcards.
+    """
+    def __init__(self, verbosity=0):
+        self.verbosity = verbosity
+
+    @staticmethod
+    def contains(mime_set, mime):
+        for part1 in mime[0], '*':
+            for part2 in mime[1], '*':
+                if (part1, part2) in mime_set:
+                    return True
+        return False
```

Comment:
I tried to copy the http [Accept header](http://tools.ietf.org/html/rfc7231#section-5.3.2) syntax, but I have changed it to fnmatch because it seems to me like it handles the accept header syntax the same but it's more powerful.